lib/fs/util.go

```diff
@@ -15,7 +15,11 @@ import (
 	"strings"
 )
 
-var errNoHome = errors.New("no home directory found - set $HOME (or the platform equivalent)")
+var (
+	errInvalidSpacePeriod  = errors.New("name ends with space or period")
+	errInvalidReservedName = errors.New("name is reserved")
+	errInvalidReservedChar = errors.New("name contains reserved character")
```

Comment:
I guess the use of word reserved is a bit vague. I'd say name contains charaters not permitted on Windows, or file name is not permitted on windows or something.

Reply:
I refactored the errors a bit to be more clear, un-exported the things as they're not used outside of lib/fs currently

Reply:
Exact wording is a bit difficult because the "name" is actually a "path component" but then it gets very technical